### DISCUSSION TITLE
[Docs] [icon] 关于 自动导入 icon 功能的正确食用方法

### DISCUSSION THREAD

**Discussion by nianwu** [element-plus/element-plus#19070] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `icon`

## Description

文档中给的示例模糊，无法正确使用el-icon相关功能, 下面提供可以使用的例子

- 使用示例 ( 并非文档中复制出来的 直接使用组件, 而是需要增加 i-ep 前缀 )
```vue
<el-icon><i-ep-Plus /></el-icon>
```

- vite.config.js 配置
```js

import IconsResolver from 'unplugin-icons/resolver'
import Inspect from 'vite-plugin-inspect'
import path from 'path'
import Icons from 'unplugin-icons/vite'

// https://vite.dev/config/
export default defineConfig({
  plugins: [
    vue(),
    AutoImport({
      imports: ['vue'],
      resolvers: [
        ElementPlusResolver(),
        // Auto import icon components
        // 自动导入图标组件
        IconsResolver({
          prefix: 'Icon'
        })
      ],
      dts: path.resolve(pathSrc, 'auto-imports.d.ts')
    }),
    Components({
      resolvers: [
        // Auto register icon components
        // 自动注册图标组件
        IconsResolver({
          enabledCollections: ['ep']
        }),
        ElementPlusResolver()
      ],
      dts: path.resolve(pathSrc, 'components.d.ts')
    }),
    Icons({
      autoInstall: true,
    }),
    Inspect(),
  ]
})

```

### Additional comments

_(empty)_

**Comment by Dsaquel** [comment]
The documentation says just that you can import an Icon or SVG and wrapp it around an <el-icon> to provide defaults styles and attributes.
Now the way your import your icons is not a concern although they talk about it ([doc](https://element-plus.org/zh-CN/component/icon.html#%E8%87%AA%E5%8A%A8%E5%AF%BC%E5%85%A5))

**Reply by nianwu** [reply]
首先ele plus 文档中是提倡使用自动导入插件的([参考](https://element-plus.org/zh-CN/guide/quickstart.html#%E8%87%AA%E5%8A%A8%E5%AF%BC%E5%85%A5-%E6%8E%A8%E8%8D%90)), 在 icon 组件中也提供了自动导入的说明，这两者会让人潜意识的认为两者的导入逻辑差不多，但是实际上官网给出的 icon 相关的自动导入示例仅提供了 vite 相关的配置, 并没有提及 使用了该配置后, 再使用 icon 组件的时候, 编写代码的方式发生了改变。
这样一来，反而违反了文档的本质，文档的作用应该是指导使用者如何使用，而不是给出一个不完整的说明。
所以要么在文档中删除相关配置提示，要么给出完整的使用示例，

**Reply by Dsaquel** [reply]
I agree that we can improve/update the doc and provide the `<i-ep-Plus>` import way.
